pkg/api/repositories.go

```diff
@@ -115,9 +115,11 @@ func (r *RepositoryUpdateRequest) FillDefaults() {
 	}
 }
 
-func (r *RepositoryRequest) FillDefaults() {
+func (r *RepositoryRequest) FillDefaults(accountID *string, orgID *string) {
 	// Currently the user cannot change these, only set at creation
 	r.ContentType = utils.Ptr(config.ContentTypeRpm)
+	r.AccountID = accountID
+	r.OrgID = orgID
```

Comment:
What prompted you to add the args here? I wonder if this would be a little confusing since accountID and orgID don't have defaults. Would we keep adding args here for fields that wouldn't have defaults? There are also other request objects with FillDefaults methods that have even fewer defaults.